opencraft/views.py

```diff
@@ -52,7 +52,7 @@ def get_redirect_url(self, *args, **kwargs):
         if InstanceReference.can_manage(user):
             return reverse('instance:index')
         else:
-            return reverse('registration:register')
+            return settings.USER_CONSOLE_FRONTEND_URL
```

Comment:
I think we should redirect users to the login page, and only redirect to the new registration form if they don't have manage permissions.